modern-phytomorphology.csl

```diff
@@ -0,0 +1,182 @@
+      </if>
+      <else>
+        <text variable="title" text-case="title" font-variant="small-caps" font-weight="bold" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=", " prefix=". ">
+      <text variable="publisher"/>
+      <text variable="publisher-place"/>
+    </group>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="page" match="none">
+        <choose>
+          <if variable="DOI">
+            <text variable="DOI" prefix="https://doi.org/"/>
+          </if>
+          <else-if variable="URL">
+            <text variable="URL"/>
+            <choose>
+              <if variable="issued" match="none">
+                <date variable="accessed" prefix=". Accessed: " suffix=".">
+                  <date-part name="month" suffix=" "/>
+                  <date-part name="day" suffix=", "/>
+                  <date-part name="year"/>
+                </date>
+              </if>
+            </choose>
+          </else-if>
+          <else-if variable="issued" match="any">
+            <text variable="status" text-case="capitalize-first"/>
+          </else-if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <citation collapse="year-suffix" et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" year-suffix-delimiter=",">
+    <sort>
+      <key variable="issued"/>
+      <key variable="author"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group font-variant="small-caps" delimiter=" ">
+        <text macro="author-short" font-variant="normal"/>
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+        <group prefix=": ">
+          <text variable="locator" prefix=" "/>
+        </group>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="19" et-al-use-first="19" et-al-subsequent-min="19" et-al-subsequent-use-first="19">
+    <sort>
+      <key macro="author-short"/>
+      <key variable="issued"/>
+    </sort>
+    <layout>
+      <text macro="author" suffix="." font-weight="bold"/>
+      <date variable="issued" prefix=" " suffix="." font-weight="bold">
+        <date-part name="year"/>
+      </date>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+          <text macro="title" prefix=" "/>
+          <text macro="editor-translator" prefix=" "/>
+          <text prefix=" " suffix="." macro="publisher"/>
+          <text macro="access"/>
+        </if>
+        <else-if type="chapter" match="any">
+          <text macro="title" prefix=" " suffix="."/>
+          <group prefix=" In: ">
+            <names variable="editor translator">
+              <name font-variant="small-caps" delimiter-precedes-last="always" et-al-min="19" et-al-use-first="19" et-al-subsequent-min="19" et-al-subsequent-use-first="19" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
```

Comment:
@novikoffav, you can delete the `et-al-min="19" et-al-use-first="19" et-al-subsequent-min="19" et-al-subsequent-use-first="19"` here, since these values carry over from `<bibliography/>`.